app/src/main/java/com/criteo/pubsdk_android/ServerBiddingActivity.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.pubsdk_android
+
+import android.os.Bundle
+import android.widget.Button
+import android.widget.LinearLayout
+import androidx.appcompat.app.AppCompatActivity
+import com.criteo.publisher.*
+import com.criteo.publisher.model.AdSize
+import com.criteo.publisher.model.BannerAdUnit
+import com.criteo.publisher.model.InterstitialAdUnit
+
+class ServerBiddingActivity : AppCompatActivity() {
+  companion object {
+    val BANNER_ADUNIT = BannerAdUnit(
+        PubSdkDemoApplication.DFP_BANNER_ADUNIT_ID,
+        AdSize(320, 50)
+    )
+
+    val INTERSTITIAL_ADUNIT = InterstitialAdUnit("/140800857/Endeavour_Interstitial_320x480")
+  }
+
+  private lateinit var criteoBannerView: CriteoBannerView
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_server_bidding)
+
+    val buttonBanner: Button = findViewById(R.id.buttonBanner)
+    buttonBanner.setOnClickListener {
+      val bannerViewContainer: LinearLayout = findViewById(R.id.bannerViewContainer)
+      criteoBannerView = CriteoBannerView(this.baseContext, BANNER_ADUNIT)
```

Comment:
The banner_adunit is useless here.
This will make two use cases with a useless parameter: In-House and server-side bidding. What do you think about adding a new constructor without ad unit ? (and with proper comment)

Reply:
Done. Though for `In-House` why isn't the AdUnit needed? I see that we are passing the AdUnit here: https://github.com/criteo/android-publisher-sdk/blob/main/app/src/main/java/com/criteo/pubsdk_android/InHouseActivity.java#L110

Reply:
For InHouse, the AdUnit is given to the `Criteo.getBidResponseFor(adUnit)`. Then the token is given to the banner view. So the ad unit given to the banner view's constructor is not really useful here.